site/content/contribute/getting-started/code-reviews.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Code Reviews"
+date: 2018-03-06T00:00:00-04:00
+weight: 5
+subsection: Getting Started
+---
+
+All changes to the product must be reviewed.
+
+* User experience changes must be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers).
```

Comment:
Propose adding `.. and may be verified by a [QA tester](/contribute/getting-started/core-committers/#qa-testers).`